app/models/user_dashboard.rb

```diff
@@ -0,0 +1,21 @@
+class UserDashboard
+  def initialize(details = {})
+    @_user = details[:user]
+  end
+
+  def fullname
+    "#{@_user.first_name} #{@_user.last_name}"
```

Comment:
Usually when I see someone use `@_user` I'll see a method wrapping the instance variable.

```ruby
def user
  @_user
end
```

And then only call directly to the `user` method in situations like this:

```ruby
def fullname
  [user.first_name, user.last_name].join(" ")
end
```